onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -806,20 +806,31 @@ void addGlobalMethods(py::module& m, Environment& env) {
         void* p_aten_op_executor = reinterpret_cast<void*>(aten_op_executor_address_int);
         contrib::aten_ops::ATenOperatorExecutor::Initialize(p_aten_op_executor);
       });
-  #ifdef ENABLE_TRAINING_TORCH_INTEROP
   m.def("register_forward_runner", [](py::object obj) -> void {
```

Comment:
What's wrong with the old code? If ENABLE_TRAINING_TORCH_INTEROP is not defined, should the python extension not have `register_forward_runner` method?

Reply:
yeah, there is no python extension. but we have some code in training python __init__.py, that need import this (but should not run it until the torch iterop feature is enabled). 

With the old code, there is a error running ortmodule training tests (without enabling torch interop)